generate_report.py

```diff
@@ -0,0 +1,41 @@
+import csv
+import sys
+
+from src.routes_report import RoutesReport
+from src.search_count_report import SearchCountReport
+
+def generate_report(filename):
+    # Make sure filename is a csv file
+    extension_idx = filename.rfind('.csv')
+    if extension_idx == -1:
+        print(f'{filename} is not a csv file! Please try again.')
+        return
+
+    # Setup reader
+    csv_file = open(filename)
+    csv_reader = csv.reader(csv_file, delimiter=',')
```

Comment:
the default delimiter is `,` according to https://docs.python.org/3.6/library/csv.html#csv-fmt-params, can we remove this everywhere since it's redundant?